src/components/organisms/PaneManager/PaneManagerLeftMenu.styled.tsx

```diff
@@ -21,3 +23,7 @@ export const Container = styled.div<{isLeftActive: boolean}>`
   gap: 7px;
   background-color: ${props => (props.isLeftActive ? BackgroundColors.darkThemeBackground : PanelColors.toolBar)}};
 `;
+
+export const SearchIcon: React.FC = () => {
```

Comment:
this could be added to the `@atoms/Icon` component

Reply:
agree!